test/acceptance/tests/basic_test.go

```diff
@@ -51,7 +51,7 @@ func TestBasic(t *testing.T) {
 			namespace := ""
 			partition := ""
 			queryString := ""
-			tfVars["consul_image"] = "public.ecr.aws/hashicorp/consul:1.12.1"
+			tfVars["consul_image"] = "public.ecr.aws/hashicorp/consul:1.15.0"
```

Comment:
Nothing exists there yet, right? Is this just optimistically hoping it will work after 1.15 is released?

Reply:
yeah, Chris and I talked about just doing this for now since it's not going to work anyways